python/cuml/cuml/accel/core.py

```diff
@@ -160,6 +161,9 @@ def install(
         `cuml.accel` accelerated for a given run.
     """
     logger.set_level(log_level)
+    # Set the environment variable if not already set so cuml.accel will
+    # be automatically enabled in subprocesses
+    os.environ.setdefault("CUML_ACCEL_ENABLED", "1")
```

Comment:
This actually makes me wonder. What do we do if `install()` is called, but `CUML_ACCEL_ENABLED` is explicitly set to "0" or "false" ? Should we just ignore that like we would currently? Issue a warning or fail?

Reply:
I think ignore (as done currently). The environment variable is a way of turning on the accelerator (and we support a falsey version for cases where changing the value but not unsetting are easier). `CUML_ACCEL_ENABLED=0` shouldn't make the other methods of invoking not work, it just disables the automatic invoking.